Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -5,6 +5,7 @@ import XCTest
 // Even if we don't run this test below OSX 10.12 we expect the actual
 // implementation to be thread safe.
 @available(OSX 10.12, *)
+@available(iOS 10.0, *)
```

Comment:
M: if we support iOS 9, why do we use 10 here?

Reply:
Because some of the APIs I use in the test here are only available since iOS 10. I use these features to check that the code, which runs also on iOS 9, is thread-safe.